nomad/config.go

```diff
@@ -389,10 +386,8 @@ type Config struct {
 	// connections from a single IP address. nil/0 means no limit.
 	RPCMaxConnsPerClient int
 
-	// LicenseConfig is a tunable knob for enterprise license testing.
+	// LicenseConfig stored information about the Enterprise license loaded for the server.
```

Comment:
```suggestion
	// LicenseConfig stores information about the Enterprise license loaded for the server.
```

I don't know how to type 🤦